code/modules/vehicle/ambulance.dm

```diff
@@ -14,20 +14,22 @@
 
 /obj/vehicle/ambulance/handle_vehicle_offsets()
 	..()
-	if(buckled_mob)
-		switch(buckled_mob.dir)
-			if(SOUTH)
-				buckled_mob.pixel_x = 0
-				buckled_mob.pixel_y = 7
-			if(WEST)
-				buckled_mob.pixel_x = 13
-				buckled_mob.pixel_y = 7
-			if(NORTH)
-				buckled_mob.pixel_x = 0
-				buckled_mob.pixel_y = 4
-			if(EAST)
-				buckled_mob.pixel_x = -13
-				buckled_mob.pixel_y = 7
+	if(buckled_mobs.len)
+		for(var/m in buckled_mobs)
```

Comment:
These won't be handled very well if more than one mob ends up buckled to a vehicle, which shouldn't happen outside of adminbuse

Reply:
```dm
if(has_buckled_mobs())
	for(var/i in 1 to buckled_mobs.len)
		var/mob/living/buckled_mob = buckled_mobs[i]
		switch(buckled_mob.dir)
			if(NORTH)
				buckled_mob.pixel_x = 0
				buckled_mob.pixel_y = 4 * i
			if(SOUTH)
				buckled_mob.pixel_x = 0
				buckled_mob.pixel_y = 7 * i
			if(EAST)
				buckled_mob.pixel_x = -13 * i
				buckled_mob.pixel_y = 7 * i
			if(WEST)
				buckled_mob.pixel_x = 13 * i
				buckled_mob.pixel_y = 7 * i